Dockerfile

```
@@ -0,0 +1,19 @@
+FROM ubuntu:16.04
+
+# ADD https://github.com/sleuthkit/scalpel/archive/master.zip /
+
+RUN apt-get update && \
+  apt-get install -y -qq --no-install-recommends \
```

```suggestion
RUN apt update && \
  apt install -y -qq --no-install-recommends \
```
it's time to let go of `apt-get`

---

apt-get should still be used in scripts